### ISSUE TITLE
PickList not accessibility conform - WCAG 2.1 AA

### ISSUE THREAD

**Issue by audiofrequenz** [primefaces/primevue#8089]
### Describe the bug

The current version of the primevue PickList is not meeting the criterie to be accessible because it has following violations to meet the criteria of the WCAG 2.1 AA

1. Hidden element focus - Focusable content should have tabindex="-1" or be removed from the DOM 

<img width="1613" height="662" alt="Image" src="https://github.com/user-attachments/assets/56121bbb-0c75-40c5-8a2b-551459b748fa" />

2. ARIA input field name - aria-label attribute does not exist or is empty, aria-labelledby does not exist and there is no title 

<img width="1602" height="697" alt="Image" src="https://github.com/user-attachments/assets/f163164b-c55a-4f65-9abf-10081a834135" />

3. Scrollable element focusable - Element should have focusable content

<img width="1602" height="872" alt="Image" src="https://github.com/user-attachments/assets/09591c50-c5a3-47d2-bc6f-58d46f84c934" />

### Reason for not contributing a PR

- [x] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/zstgbahx?file=src%2FApp.vue

### Environment

Tested in own project as well as a Storybook with axe DevTools. Is also visible on privevue.org/picklist/ with chrome Lighthouse. But should be a problem in all projects containing a PickList. Just serve and inspect with lighthouse or any other tool

### Vue version

3.5.20

### PrimeVue version

4.3.8

### Node version

22.14.0

### Browser(s)

Chrome 140.0.7339.81

### Steps to reproduce the behavior

1. Serve any Project on localhost with a PickList
2. Use Chrome to display site (with axe DevTools for example)
3. Right click -> Inspect
4. Lighthouse or axe DevTools
5. Site Scan your localhost for accessibility

### Expected behavior

There should not be any accessibility related problems with the PickList-Component related to wcag 2.1 or 2.2 AA
